packages/dashboard-frontend/src/services/bootstrap/index.ts

```diff
@@ -237,11 +249,7 @@ export default class Bootstrap {
     try {
       await requestNamespaces()(this.store.dispatch, this.store.getState, undefined);
     } catch (e) {
-      this.appAlerts.showAlert({
-        key: 'bootstrap-request-namespaces',
-        title: common.helpers.errors.getMessage(e),
-        variant: AlertVariant.danger,
-      });
+      console.error(e);
```

Comment:
Should we have at least one general error alert in the case when preload data is failed?